tests/unit/plugins/test_gcp_utils.py

```diff
@@ -143,3 +143,34 @@ def test_arrays_dicts_with_difference(self):
         request2 = GcpRequest(value2)
         self.assertNotEquals(request1, request2)
         self.assertEqual(request1.difference(request2), difference)
+
+    def test_dicts_boolean_with_difference(self):
```

Comment:
is it also worth testing with strings too? since "to_text" is used.

Reply:
Could you clarify the use case, what will we be testing? Dictionaries of strings are tested in previous tests.

Reply:
ok, great! that's what I was referring to.